licensed-software.shtml

```diff
@@ -122,11 +138,14 @@ a submit file for jobs that use our licensed software modules.
 They are shown in the sample submit file below and include: </p>
 
 <ul>
-	<li><b>Require access to the modules.</b> There is a special requirements statement
-	that ensures a computer has access to Gluster (where the modules live) and 
-	that the module command is working. </li>
+	<li><b>Require access to the modules.</b> To ensure that your job will have access 
+	to CHTC software modules you must include the following in your submit file.
+	<pre class="sub">
+requirements = (HasCHTCSoftware == true)</pre></li>
```

Comment:
This attribute is called "HasChtcSoftware" (case difference) in the example job below.  Match the two?